core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -5642,7 +5644,38 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
 #endif
                 goto call_func_from_entry;
             }
+#if WASM_ENABLE_SIMD != 0
+            HANDLE_OP(WASM_OP_SIMD_PREFIX)
+            {
+                GET_OPCODE();
+
+                switch (opcode) {
+                    case SIMD_v128_const:
+                    {
+                        uint8 *orig_ip = frame_ip;
+
+                        frame_ip += sizeof(V128);
+                        addr_ret = GET_OFFSET();
 
+                        bh_memcpy_s(frame_lp + addr_ret, sizeof(V128), orig_ip,
+                                    sizeof(V128));
```

Comment:
Not sure whether using memcpy may impact the performance, maybe it is better if we define the macro like GET_V128_FROM_ADDR and PUT_V128_TO_ADDR in wasm_runtime_common.h, and it should be needed when more opcodes are implemented.

Reply:
Hmm, the other I*_CONST opcodes were implemented using bh_memcpy_s, but I don't mind doing it differently. Updated the code.